### DISCUSSION TITLE
What does MCF stand for?

### DISCUSSION THREAD

**Discussion by AdreKiseque** [lhmouse/mcfgthread#262] [General]
I've looked everywhere and can't find anything. Is it just an arbitrary name?

**Comment by lhmouse** [comment]
**Yes.**

Originally it meant _Mouse C++ Framework_, later was changed to _Mouse C++ Foundation_, and later was changed to _Meta-purpose C++ Foundation_. However there has never been a meaning attached to _M_; please deem it cyber-mysticism.

Despite the origination about _C++_, mcfgthread is actually implemented in pure C.
